### PR TITLE
Remove DiscoverBroker RPC and add ZKClient

### PR DESCRIPTION
**Pull request by elon0823**
related to https://github.com/paust-team/shapleq/pull/131, https://github.com/paust-team/shapleq/pull/135

### CODE REVIEW

File: shapleqclient/admin.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def __init__(self, config: QConfig, logger: logging.Logger):
         self.logger = logger
 
     def setup(self):
-        self._client.connect_to_broker(self._client.config.get_broker_address(), self._client.config.get_broker_port())
+        self._client.connect_to_broker(self._client.config.get_bootstrap_servers())
```
Comment (1dennispark) [inline shapleqclient/admin.py:22]: We need specify keyword that one of both `bootstrap servers` and `zk quorum`

File: test/stream_test.py [hunk 2]
```diff
@@ -15,8 +15,7 @@
 
 class StreamTest(unittest.TestCase):
     zk_host = "127.0.0.1:2181"
-    broker_port = 1101
-    broker_address = "127.0.0.1"
+    broker_host = "127.0.0.1:1101"
```
Comment (1dennispark) [inline test/stream_test.py:18]: ditto.

Comment (elon0823) [reply test/stream_test.py:18]: That line is nothing to do with the above comment. It simply represents the broker host as a string of the {ip:port} format.

File: shapleqclient/admin.py (hunk 1 above)
Comment (elon0823) [reply shapleqclient/admin.py]: Fixed

**Review by 1dennispark — APPROVED** [review]

**Pull request merged**